README.md

```diff
@@ -41,15 +41,15 @@ pg-schema-diff plan --dsn "postgres://postgres:postgres@localhost:5432/postgres"
 - Online index replacement
 
 # Install
-## Library
-```bash
-go get -u github.com/stripe/pg-schema-diff
-````
 ## CLI
 ```bash
```

Comment:
People are probably more interested in the CLI than the library, so let's put that first